packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,313 @@
+      ? transparentize(0.3, props.theme.colors.neutral100)
+      : props.theme.colors.neutral500};
+  div > div {
+    margin: 4px 0 0 0;
+  }
+  span {
+    display: flex;
+    text-align: left;
+  }
+`;
+
+// Stepper!
+export const Stepper = React.forwardRef<HTMLDivElement, StepperProps>(
+  (props, ref) => {
+    const {
+      breakpoint,
+      breakpointStyle,
+      children,
+      currentStep,
+      stepDescriptionLabel,
+      isInverse: isInverseProp,
+      hideLabels,
+      summaryView,
+      testId,
+      ...rest
+    } = props;
+    const theme = React.useContext(ThemeContext);
+    const isInverse = useIsInverse(isInverseProp);
+
+    let [breakPointHideLabels, setBreakPointHideLabels] = React.useState(false);
+    let [breakPointSummaryView, setBreakPointSummaryView] =
+      React.useState(false);
+
+    // Breakpoint states when a specific breakpoint number is set, then a breakpointStyle which results in the removal of all text or the summary view.
+    const [windowWidth, setWindowWidth] = React.useState(window.innerWidth);
+
+    const hasBreakpoint = breakpoint > windowWidth;
+    const beyondBreakpoint = breakpoint < windowWidth;
+
+    React.useEffect(() => {
+      function handleResize() {
+        setWindowWidth(window.innerWidth);
+      }
+      window.addEventListener('resize', handleResize);
+      handleResize();
+
+      if (hasBreakpoint && breakpointStyle === BreakPointStyle.summaryView) {
+        setBreakPointSummaryView(true);
+      } else if (
+        hasBreakpoint &&
+        breakpointStyle === BreakPointStyle.hideLabels
+      ) {
+        setBreakPointHideLabels(true);
+      } else if (
+        (beyondBreakpoint && breakpointStyle === BreakPointStyle.hideLabels) ||
+        (beyondBreakpoint && BreakPointStyle.summaryView)
+      ) {
+        setBreakPointSummaryView(false);
+        setBreakPointHideLabels(false);
+      }
+
+      return () => window.removeEventListener('resize', handleResize);
+    }, []);
+
+    // Sets aria-current based on the active step, else false.
+    const ariaCurrent = index => {
+      if (currentStep === index) {
+        return true;
+      }
+      return false;
+    };
+
+    // Step states
+    const steps = React.Children.toArray(children);
+
+    const Steps = React.Children.map(
+      children,
+      (
+        child: React.ReactElement<React.PropsWithChildren<StepProps>>,
+        index
+      ) => {
+        if (child.type === Step) {
+          const stepStatusStyles =
+            currentStep >= index + 1
+              ? StepStatus.complete
+              : currentStep >= index
+              ? StepStatus.active
+              : StepStatus.incomplete;
+
+          const item = React.cloneElement(child, {
+            'aria-current': ariaCurrent(index),
+            children: currentStep === index ? child : null,
+            key: index,
+            isInverse: isInverse,
+            hideLabels: breakPointHideLabels ? true : hideLabels,
```

Comment:
This is equivalent to:
```suggestion
            hideLabels: breakPointHideLabels || hideLabels
```